src/main/java/nl/altindag/log/LogCaptor.java

```diff
@@ -193,13 +208,13 @@ public void resetLogLevel() {
     }
 
     public void clearLogs() {
-        listAppender.list.clear();
+        eventList.clear();
```

Comment:
Should this als be wrapped in a synchronize block?

Reply:
clear() is synchronized internally

Reply:
Ah, yes it is obviously using the syncronized list and therefor it is safe to call. Sorry missed that one